lib/recipient.rb

```diff
@@ -0,0 +1,43 @@
+require 'httparty'
+require 'dotenv'
+require_relative 'slack_api_error'
+
+Dotenv.load
+
+module Slack
+  class Recipient
+    
+    attr_reader :name, :id
+    
+    def initialize(name: , id: )
+      @name = name
+      @id = id
+    end
+    
+    def self.get(url, params)
+      response = HTTParty.get(url, query: params)
+      
+      if response.code != 200 || response['ok'] == false
+        raise Slack_Api_Error.new "Error! No response found."
```

Comment:
Including `response["error"]` in the error message would have been nice.  Also, the convention is not to use `_`s in class names.

```suggestion
        raise SlackApiError.new "Error! No response found #{response["error"]}."
```